sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -22,6 +26,11 @@ internal class RadarActivityLifecycleCallbacks : Application.ActivityLifecycleCa
         }
         count++
         foreground = count > 0
+
+        if (ContextCompat.checkSelfPermission(activity.applicationContext, Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_DENIED &&
+            !ActivityCompat.shouldShowRequestPermissionRationale(activity, Manifest.permission.ACCESS_FINE_LOCATION)) {
+            RadarSettings.setPermissionsDenied(activity.applicationContext, true)
+        }
     }
 
     override fun onActivityPaused(activity: Activity) {
```

Comment:
is `onActivityResumed` the only place where we need the condition? seems like we have other activity handlers that take an `Activity` as well